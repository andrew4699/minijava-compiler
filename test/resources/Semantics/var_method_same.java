class a {

  public static void main(String[] args) {
    System.out.println(new b().valid());
  }
}
class b {
  int valid;
  public int valid() {
    valid = 1;
    return valid;
  }
}