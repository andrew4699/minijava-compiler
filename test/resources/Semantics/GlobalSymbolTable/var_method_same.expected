Global Symbol Table:

Class: a
  Superclass: None

  Fields:

  Methods:
    void main(String[] args)

Class: b
  Superclass: None

  Fields:
    int valid

  Methods:
    int valid()
